efficientdet/coco_metric.py

```diff
@@ -148,19 +149,19 @@ def update_state(self, groundtruth_data, detections):
         [image_id, x, y, width, height, score, class].
     """
     for i in range(len(detections)):
+      detections_i = detections[i]
```

Comment:
It looks like i is only used to index into detections and groundtruth.  Maybe replace the for loop with:

```
for d, gt in zip(detections, groundtruth:
```